resources/sass/pages/_home.scss

```diff
@@ -0,0 +1,104 @@
+#categories-id {
+    .home-section {
+        padding: 33px 0px;
+        max-width: 1140px;
+        margin: 0 auto;
+        position: relative;
+        .row {
+            @include div-row;
+            text-align: center;
+            flex-direction: row;
+        }
+        &:nth-child(odd) {
+            .row {
+                flex-direction: row-reverse;
+            }
+        }
+        .offer {
+            display: none;
+            width: 100%;
+        }
+        .w3-transparent {
+            background-color: #7f7f7f !important;
+        }
+        .w3-left,
+        .w3-right {
+            cursor: pointer
+        }
+        .w3-badge {
+            height: 7px;
+            width: 7px;
+            padding: 3px;
+            margin: 1%;
+            cursor: pointer
+        }
+        .w3-grey {
+            background-color: #999999 !important;
+            opacity: 0.6;
+            font-weight: 400;
+        }
+        .w3-button {
+            color: $color_white !important;
+            &:hover {
+                color: $color_white !important;
+            }
+        }
+        .w3-content {
+            max-width: 1150px !important;
+        }
+        .cat_button {
+            background: $color_red;
+            color: $color_white;
+            border-color: $color_red;
+            padding: 1%;
+            font-weight: 500;
+            font-size: 1em;
+            /* line-height: 2%; */
+        }
+        .cat-inner-heading {
+            font-family: $primary_font !important;
```

Comment:
What is this deal with !important

Reply:
I have already removed !important from everywhere.